databricks_cli/pipelines/cli.py

```diff
@@ -236,6 +266,13 @@ def _validate_pipeline_id(pipeline_id):
         error_and_quit(message)
 
 
+def _handle_duplicate_name_exception(spec, exception):
+    if json.loads(exception.response.text).get('error_code', "") == 'RESOURCE_CONFLICT':
```

Comment:
Is this safe to do? It's JSON if it's a DatabricksServiceException, but I'm not sure it's guaranteed to be?

Reply:
That's true. Added some additional error handling to account for this.